app/views/trainees/index.html.erb

```diff
@@ -1,6 +1,9 @@
 <%= render PageTitle::View.new(title: trainees_page_title(@paginated_trainees)) %>
 
-<h1 class="govuk-heading-xl">Trainee records</h1>
+<% if @paginated_trainees.current_page > 1 %>
+  <span class="govuk-caption-xl">Page <%= @paginated_trainees.current_page %> of <%= @paginated_trainees.total_pages %></span>
+<% end %>
+<h1 class="govuk-heading-xl">Trainee records (<%= @filtered_trainees.count %><span class="govuk-visually-hidden"> <%= "record".pluralize(@filtered_trainees.count) %></span>)</h1>
```

Comment:
Similar to where we have `@draft_trainees_count` and `@completed_trainees_count`, let's define a new instance variable `@total_trainees_count` in the controller and use it wherever you have `@filtered_trainees.count`.

This way, you're performing the `.count` calculation just once and moving the code out of the view.